extensions/chocolatey-core.extension/extensions/Get-AvailableDriveLetter.ps1

```diff
@@ -0,0 +1,40 @@
+﻿<#
+.SYNOPSIS
+  Get a 'free' drive letter
+
+.DESCRIPTION
+  Get a not yet in-use drive letter that can be used for mounting
+
+.EXAMPLE
+  Get-AvailableDriveLetter
+
+.EXAMPLE
+  Get-AvailableDriveLetter 'X'
+  (do not return X, even if it'd be the next choice)
+
+.INPUTS
+  specific drive letter 
```

Comment:
Even though the parameter is self explanatory, this should probably say that it's the drive letter(s) that its not to use.